### PR TITLE
Show warning message if file not found and don't fail entire operation

### PR DESCRIPTION
**Pull request by chintanrp**

### CODE REVIEW

File: pyServer/GuestFishWrapper.py [hunk 1]
```diff
@@ -205,23 +205,25 @@ def RunCredentialScanner(self, targetDir, operationOutFile):
...
+                                f.seek(0)
+                                for i, line in enumerate(lines):
+                                    if (i+1) != int(line_number):
+                                        f.write(line)
+                                    else:
+                                        f.write("***REDACTED***")
+                                f.truncate()
+                        else:
+                            os.remove(source_file)
+                except:
```
Comment (CraigWiand) [inline pyServer/GuestFishWrapper.py]: This should be of type `OSError`, right?  Perhaps we can catch that type and log the actual error being thrown?

Comment (chintanrp) [reply pyServer/GuestFishWrapper.py]: Yes, that's the good idea. Let's catch the type and log the actual error. Which will me more specific then just handling OSError

Comment (waynekuo) [reply pyServer/GuestFishWrapper.py]: I see you're capturing the base Exception type and printing the exception now. Do you know when there's a file related error, what the stringified output will be? Will this include the file name, etc? 

The benefit of catching OSError is you have more specific properties to use for message output. See: https://docs.python.org/3/library/exceptions.html#OSError

Comment (chintanrp) [reply pyServer/GuestFishWrapper.py]: For file related error, sample message will be like below
"CredentialScanner: Failed to run due to [Error] No such file or directory: 'filename'

File: pyServer/GuestFishWrapper.py [hunk 2]
```diff
@@ -205,23 +205,25 @@ def RunCredentialScanner(self, targetDir, operationOutFile):
...
-                                else:
-                                    f.write("***REDACTED***")
-                            f.truncate()
-                    else:
-                        os.remove(source_file)
-
+                try:
+                    # Remove secret - delete line for small files, entire file for large files
+                    if os.path.isfile(source_file):
+                        if os.path.getsize(source_file) < 1000000:
```
Comment (waynekuo) [inline pyServer/GuestFishWrapper.py]: LGTM

**Pull request merged**